core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -73,6 +73,28 @@ coreos:
 
         [Install]
         RequiredBy=format-etcd2-volume.service
+{{if .AmazonSsmAgent.Enabled}}
+    - name: amazon-ssm-agent.service
+      command: start
+      enable: true
+      content: |
+        [Unit]
+        Description=amazon-ssm-agent
+        Requires=network-online.target
+        After=network-online.target
+
+        [Service]
+        Type=simple
+        WorkingDirectory=/opt/bin/ssm
```

Comment:
Yes, as I tested and confirmed. `aws ssm send-command` result in running commands in this working directory, in this case `/opt/bin/ssm`. However, you can set working directory with `aws ssm send-command`.

Anyhow, I am writing a fix for this and will commit it once I am done with testing.